src/cpp/Makefile

```
@@ -1,3 +1,19 @@
+ # 
+ # SPDX-License-Identifier: GPL-3.0-only
+ # 
+ # Copyright 2008-2024 San Diego State University (SDSU) and Contributors 
```

Hi @aneeshs1729 , thank you so much for your contribution! 

We need the following line:
```suggestion
 Copyright 2008-2024 San Diego State University Research Foundation (SDSURF).
```

Please change it in all fines. Thank you!

---

Hello @valeriabarra 
I took your change to heart and changed all the copyright info to all of the files

---

Makefile still has the old reference to "contributors"